scene/gui/control.cpp

```diff
@@ -1662,6 +1668,11 @@ void Control::set_position(const Size2 &p_point, bool p_keep_margins) {
 }
 
 void Control::_set_size(const Size2 &p_size) {
+#ifdef DEBUG_ENABLED
+	if (data.size_warning) {
+		WARN_PRINT("Setting the size of Control inside _ready() is unreliable. Consider deferring it with set_deferred().");
```

Comment:
Note that this will also be printed for stuff that are before ready (`_init`, `_enter_tree`), so the message might be misleading.

Reply:
How should it be changed then? `Setting the size of Control before it's fully initialized`?

Reply:
Adjusting the size of Controll node at startup time is unreliable ... ?

Reply:
Is "startup" a precise term though? Someone might understand it as app startup.

Reply:
Adjusting the size of not initialized Control node is unreliable. ... ?